docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -111,3 +107,22 @@ Finally, establish a connection by attaching a GSM antenna (Rubber ducky) with a
 
 To check if you established a connection, you can use the `meadow listen` CLI command, which should return a message like this:
 `Connection established successfully! IP address '100.69.106.222'.`
+
+#### Handling Cell connection using a .NET application
```

Comment:
Could you check if it's a good example of the `IsConnected` property usage? @alexischr

Reply:
Let's make `while(cell.IsConnected)` to `if(cell.IsConnected)` (to avoid a tight loop with no sleep, especially one that spams text), and add an `else` for the 'not connected' case.  This way your example is a small function that tells you every second if you are connected or not. That is good enough!